packages/babel-parser/src/parser/statement.js

```diff
@@ -1911,6 +1925,17 @@ export default class StatementParser extends ExpressionParser {
       if (isDefault) {
         // Default exports
         this.checkDuplicateExports(node, "default");
+        if (this.hasPlugin("exportDefaultFrom")) {
+          const declaration = ((node: any): N.ExportDefaultDeclaration)
+            .declaration;
+          if (
+            declaration.type === "Identifier" &&
+            declaration.name === "from" &&
+            declaration.end - declaration.start === 4 // does not contain escape
```

Comment:
`export default (from)` is allowed in https://tc39.es/proposal-export-default-from/#prod-ExportDeclaration